targets/csharp/templates/PlayFabUtil.cs.ejs

```diff
@@ -49,6 +51,100 @@ namespace PlayFab
                 return null;
             return error.GenerateErrorReport();
         }
+
+#if NET45 || NETCOREAPP2_0
+        [ThreadStatic]
+        private static StringBuilder _sb;
+        /// <summary>
+        /// A threadsafe way to block and load a text file
+        /// 
+        /// Load a text file, and return the file as text.
+        /// Used for small (usually json) files.
+        /// </summary>
+        public static string ReadAllFileText(string filename)
+        {
+            if (!File.Exists(filename))
+                return string.Empty;
+                
+            if (_sb == null)
+                _sb = new StringBuilder();
+            _sb.Length = 0;
+
+            FileStream fs = null;
+            try 
```

Comment:
I don't understand why we have the try/finally thing rather than just having two using blocks, one for FileStream and one for BinaryReader.

Reply:
Above comment for braces answers this. I'll update it with our using pattern.